geode-docs/tools_modules/gfsh/command-pages/alter.html.md.erb

```diff
@@ -226,17 +226,17 @@ alter region --name=value [--groups=value(,value)*]
 </tr>
 <tr class="even">
 <td><span class="keyword parmname">\-\-cache-listener</span></td>
-<td>Fully qualified class name of a plug-in to be instantiated for receiving after-event notification of changes to the region and its entries. Any number of cache listeners can be configured.</td>
+<td>Fully qualified class name of a plug-in to be instantiated for receiving after-event notification of changes to the region and its entries. Any number of cache listeners can be configured. A fully qualified class name may be appended with a JSON specification that will be parsed to become the fields of the parameter to the <code>init()</code> method for a class that implements the <code>Declarable</code> interface.</td>
```

Comment:
do we give an example somewhere in the docs? I think it would be more visual if we have an example. Also would be good to call out that the json can have singe quotes or double quotes.

Reply:
We do not yet have the examples/explanation.  That task will be done with GEODE-4737.